apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -1,6 +1,6 @@
 "use client";
 
-import { useAutoAnimate } from "@formkit/auto-animate/react";
+import { Reorder } from "framer-motion";
```

Comment:
I guess Reorder already has a built-in animation so we don't need useAutoAnimate when changing through buttons